sirepo/template/jspec.py

```diff
@@ -583,3 +529,77 @@ def _set_mass_and_charge(ion_beam):
         ion_beam.mass, ion_beam.charge_number = _SPECIES_MASS_AND_CHARGE[
             ion_beam.particle
         ]
+
+
+def _v2_add_report_mtime(result, model_name, path):
+    if path.exists():
+        result.append(
```

Comment:
Let's remove result. It's not the right concept here. This should just be reports.